src/main/java/ch/njol/skript/lang/VariableString.java

```diff
@@ -631,61 +623,61 @@ public StringMode getMode() {
 		return mode;
 	}
 
+	@SuppressWarnings("resource")
 	public VariableString setMode(StringMode mode) {
 		if (this.mode == mode || isSimple)
 			return this;
-		@SuppressWarnings("resource")
-		BlockingLogHandler h = new BlockingLogHandler().start();
+		BlockingLogHandler blockLogs = new BlockingLogHandler().start();
```

Comment:
This can be a try-with-resources, right?

Reply:
No, I don't think so, because it needs to be available in the `finally`.

Reply:
Ah it is only used in the finally block to stop the resource. Since it is an AutoCloseable, using the resource in a try-with-resources block will automatically close the resource when the block is exited.

Reply:
Oh that's perfect than, thanks